### PR TITLE
JBPM-8361- Create on-demand Spring Boot PR check

### PR DESCRIPTION
**Pull request by MarianMacik**

### CODE REVIEW

**Comment by MarianMacik** [comment]
jenkins execute springboot build

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins execute springboot build

**Comment by MarianMacik** [comment]
jenkins execute springboot build

**Comment by MarianMacik** [comment]
jenkins execute springboot build

**Comment by MarianMacik** [comment]
jenkins execute springboot build

**Comment by MarianMacik** [comment]
jenkins execute springboot build

**Comment by MarianMacik** [comment]
jenkins execute springboot build

**Comment by MarianMacik** [comment]
@mswiderski Spring Boot on-demand job is done, are you ok with the current config?

File: pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
   <name>Drools and jBPM integration multiproject</name>
   <description>
-    Drools and jBPM integration with seam, spring, camel, ...
+    Drools and jBPM integration with seam, spring, camel, ....
```
Comment (mswiderski) [inline pom.xml]: come on 4 dots instead of three ... that's unacceptable!! :)

Comment (MarianMacik) [reply pom.xml]: I am going to the shame corner right now...

**Comment by MarianMacik** [comment]
jenkins execute springboot build

**Pull request merged**